independent-publisher-connectors/PaySpace/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4476 @@
+            "x-ms-summary": "{{BankDetailId}}"
+          },
+          {
+            "name": "CustomAuthHeader",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "description": "Bearer token value (without the keyword Bearer)",
+            "x-ms-summary": "Bearer token"
+          },
+          {
+            "name": "CustomEnvironmentHeader",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "description": "Variable that is either api (for PROD) or apistaging (for STAGING)",
+            "x-ms-summary": "Environment Variable"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "type": "string"
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {}
+            }
+          }
+        }
+      }
+    },
+    "/odata/v1.1/{company-id}/$metadata": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Metadata",
+        "description": "The response is a list of available entities that can be requested as well as enumeration values. The response will differ based on your permissions.  Note: Metadata can change depending on company settings and permissions",
+        "operationId": "GetMetadata",
+        "parameters": [
+          {
+            "name": "company-id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "{{company-id}} The Company Id",
+            "x-ms-summary": "{{company-id}}"
+          },
+          {
+            "name": "CustomAuthHeader",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "description": "Bearer token value (without the keyword Bearer)",
+            "x-ms-summary": "Bearer token"
+          },
+          {
+            "name": "CustomEnvironmentHeader",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "description": "Variable that is either api (for PROD) or apistaging (for STAGING)",
+            "x-ms-summary": "Environment Variable"
+          }
+        ]
+      }
+    }
+  },
+  "definitions": {},
+  "parameters": {},
+  "responses": {},
+  "securityDefinitions": {},
+  "security": [],
+  "tags": [],
+  "x-ms-connector-metadata":[
+    {
+        "propertyName": "Website",
+        "propertyValue": "https://www.payspace.com"
+    },
+    {
+        "propertyName": "Privacy policy",
+        "propertyValue": "https://www.payspace.com/privacy-policy/;https://za.mintgroup.net/privacy-policy/"
```

Comment:
@mint-virgil please include the stackowner privacy policy page only. Thank you

Reply:
You can add yours in the readme file :-).

Reply:
@Amjed-Ayoub - noted on the responses - I'll look into that now.
Thank you very much

Reply:
updated in latest commit